frontend/src/views/web/challenge/edit-challenge/edit-challenge-evaluation-criteria.html

```diff
@@ -21,7 +21,8 @@
                                      trix-attachment-remove="trixAttachmentRemove(e, editor);"
                                      prevent-trix-file-accept = "true"
                                      placeholder="Evaluation Criteria">
-                          </trix-editor>
+                          <trix-editor>
+                        <code>Hint: To add Mathjax use $$....mathjax code...$$</code> 
```

Comment:
What does this look like?

Reply:
Make this "To use MathJax in editor, use $$ <your mathjax code> $$". Use escape characters if needed.

Reply:
Make the same change everywhere.